MapboxNavigationTests/CocoaPodsTest/PodInstall/Podfile

```diff
@@ -0,0 +1,9 @@
+platform :ios, '9.0'
+use_frameworks!
+
+target 'PodInstall' do
+  pod 'MapboxGeocoder.swift', '~> 0.6.0'
+  pod 'MapboxNavigationUI.swift', :git => 'https://github.com/mapbox/MapboxNavigation.swift.git', :branch => 'BRANCH_NAME'
+  pod 'MapboxNavigation.swift', :git => 'https://github.com/mapbox/MapboxNavigation.swift.git', :branch => 'BRANCH_NAME'
```

Comment:
This threw me for a loop. A comment to the effect of https://github.com/mapbox/MapboxNavigation.swift/pull/54#discussion_r103747252 would be useful here.